h/static/styles/partials-v2/_pager.scss

```diff
@@ -0,0 +1,36 @@
+.pager {
+  list-style: none;
+  display: flex;
+  flex-direction: row;
+  margin-top: 45px;
+  font-weight: bold;
+  justify-content: center;
+}
+
+.pager__item {
+  border-radius: 2px;
+  color: $grey-6;
+  padding: 7px 10px 7px 10px;
+}
+
+.item-highlight {
+  background-color: #ececec;
+}
+
+.item-begin {
```

Comment:
You could use `:first-of-type` and `:last-of-type` or `:first-child` / `:last-child` pseudo-element selectors here rather than introducing a new class.